lunchbot.py

```diff
@@ -154,13 +172,41 @@ def handle_lunchbot_command(self, command, user):
                 self.update_data()
                 self.read_data()
             else:
-                post_ephemeral = True
+                self.post_ephemeral = True
                 msg = 'Uh oh! %s is not in the list of restaurants.' % restaurant
+        except Exception as e:
+            self.post_ephemeral = True
+            msg = 'Please try again.\n%s' % e
+        return msg
+
+    def handle_lunchbot_command(self, command, user):
+        """
+        Takes a command from Slack chat directed at Lunchbot and performs the action if it is supported
+        """
+        self.post_ephemeral = False
+
+        if 'help' in command.lower():
+            self.post_ephemeral = True
+            msg = self.HELP_TEXT
+        elif 'list restaurants' in command.lower():
+            self.post_ephemeral = True
+            msg = self.BLANK
+            for restaurant in self.RESTAURANTS:
+                msg = '\n'.join([msg, '%s has weight %s' % (restaurant['name'], restaurant['weight'])])
+        elif 'add restaurant' in command.lower():
+            msg = self.add_restaurant_command(command)
+        elif 'remove restaurant' in command.lower():
+            msg = self.remove_restaurant_command(command)
+        elif '' in command.lower():
```

Comment:
What the difference between this case and the else case?

Reply:
this is if its a blank...

like lunchbot <blank>
else if its anything other than blank

Reply:
lucnhbot sushi